evaluation/tasks/piqa/piqa.py

```diff
@@ -0,0 +1,83 @@
+# Module for any additional processing required for the TyDi QA dataset
+# HuggingFace dataset link: https://huggingface.co/datasets/piqa
+from datasets import load_dataset
+from jinja2 import Template
+from torch.utils.data import Dataset
+from tqdm import tqdm
+
+from evaluation.tasks.auto_task import AutoTask
+
+
+TEMPLATE = Template(
+    """
+Given a goal and 2 solutions, choose the most appropriate solution.
+Goal: {{goal}}
+{{'Solution 1'}}: {{sol1}}
+{{'Solution 2'}}: {{sol2}}
+Answer:
+    """
+)
+
+
+class PIQADataset(Dataset):
+    def __init__(self, tokenizer):
+        super().__init__()
```

Comment:
Please correct me if I'm wrong, but wouldn't this fail because there is no JSON configuration file? Either we should modify `AutoTask.load_task_args()` to account for non-existent configuration files, or add an empty JSON file in the `piqa` directory titled `english.json. What do you think?

Reply:
Yes, I think adding an empty JSON file is the easy fix for now. We could have a template task folder, specifying some code format and the required files under each task folder, e.g. always having an `english.json`, and having a `multilingual.json` if the task is multilingual, etc.

Reply:
Makes sense. Yes, I assume we will have a template folder at some point that has the template code along with the empty JSON files. Thanks for the quick fix!